### PR TITLE
Tt 3096 new plugin

### PR DESCRIPTION
**Pull request by jmortlock**
### Why  
Extract this custom plugin out of Quicktravel and into a package that can be pulled in by webpack

### CODE REVIEW

File: tags/dialogs/tags.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      }
+    ],
+    onShow: function() {
+      var element, selection;
+      selection = editor.getSelection();
+      element = selection.getStartElement();
+      if (element) {
+        element = element.getAscendant("abbr", true);
+      }
+      if (!element || element.getName() !== "abbr") {
```
Comment (HellRok) [inline tags/dialogs/tags.js]: I don't think we can ever get to the second part of this `or`

File: README.md [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
+## ckeditor-liquidtag-plugin  
 
+Adds a toolbar icon and dialogue which allows the user to select a record from a pre-defined list of records.
 
 ### How to Install?
 
-```
-npm -i ckeditor-tags-plugin --save```
+```npm -i ckeditor-tags-plugin --save```
+
```
Comment (HellRok) [inline README.md]: Accidental double new line?

**Comment by mnoack** [comment]
I think it's ready to merge (I'm fairly happy to accept the code is presumably a copy of what is in our live system anyway and merge on that basis alone.

Maybe just quickly chat to Sean before merging.

**Comment by jmortlock** [comment]
Interestingly I have found a tutorial / sample code that explains how to setup plugin tests for ckeditor, so I will have a look at this before merging

**Comment by jmortlock** [comment]
Code coverage is now 100% + I added linting for good measure.

**Comment by jmortlock** [comment]
Also changed the behaviour previously the editor would insert the selection inside an <abbr> tag, however we really only want the raw text i.e. before

```<abbr>{{reservation.name}}</abbr>```

After
```{{reservation.name}}```

This simplifies the code and gives a better end result for the user.

**Pull request merged**